frontend/catalyst/jit.py

```diff
@@ -430,6 +433,21 @@ def sum_abstracted(arr):
     if fn is None:
         return functools.partial(qjit, **kwargs)
 
+    # Map static_argnames to static_argnums
+    # no need to propagate static_argnames further
+    if static_argnames is not None:
```

Comment:
What is Jax behaviour when you give an argname that is not part of the function?

Reply:
```python
@partial(jax.jit, static_argnames=('xx',))
def circuit(x,y):
    if x<10:
        return x+y

res = circuit(1, 2)
print(res)
```

```
ValueError: Jitted function has invalid argnames {'xx'} in static_argnames. Function does not take these args.
```

I will add this to qjit